packages/ndla-ui/src/Article/ArticleByline.tsx

```diff
@@ -59,6 +59,7 @@ type Props = {
   licenseBox?: ReactNode;
   locale?: string;
   footnotes?: FootNote[];
+  isFrontpageArticle?: boolean;
```

Comment:
Eg såg heller for meg prop: accordionHeaderVariant: 'blue' | 'white'. Då er det ikkje knytta til type artikkel og kan brukes litt meir generellt.

Reply:
Ja! Det høres my bedre ut!